triggers/messageReply.ts

```diff
@@ -0,0 +1,185 @@
+import { ZObject, Bundle } from "zapier-platform-core";
+import {
+  generateInputFields,
+  transformInputData,
+  InputBundle,
+} from "../api/apis";
+import { addHeaders } from "../authentication";
+import { v4 as uuidv4 } from "uuid";
+
+const messageReplyCallbackCreateInput = [
+  {
+    key: "name",
+    required: true,
+    label: "Name",
+    helpText: "The name of the callback (make sure to provide a unique value)",
+  },
+  {
+    key: "removeHTML",
+    required: false,
+    label: "RemoveHTML",
+    helpText: "Specifies whether HTML should be stripped from responses.",
+    choices: {
+      enabled: "enabled",
+      disabled: "disabled",
+    },
+  },
+  {
+    key: "retriesEnabled",
+    required: false,
+    label: "Retries Enabled",
+    helpText:
+      "Specifies whether Whispir should perform retries in the event there is a failure accessing the callback service.",
+    choices: {
+      true: true,
+      false: false,
+    },
+  },
+  {
+    key: "email",
+    required: true,
+    label: "Email",
+    helpText:
+      "Whispir notify this email address that a callback has failed (only when retries are disabled)",
```

Comment:
Can we update to `notifies` and `when`

`"Whispir notifies this email address when a callback has failed (only when retries are disabled)",`

Reply:
yes, I'll update this. Thanks sharie for pointing it out